render/src/bitmap.rs

```diff
@@ -9,12 +11,19 @@ pub struct BitmapInfo {
     pub height: u16,
 }
 
+#[derive(Copy, Clone, Debug)]
+pub struct BitmapSourceInfo {
+    pub width: u16,
+    pub height: u16,
+}
+
 /// An object that returns a bitmap given an ID.
 ///
 /// This is used by render backends to get the bitmap used in a bitmap fill.
 /// For movie libraries, this will return the bitmap with the given character ID.
 pub trait BitmapSource {
-    fn bitmap(&self, id: u16) -> Option<BitmapInfo>;
+    fn bitmap(&self, id: u16) -> Option<BitmapSourceInfo>;
```

Comment:
Same here; if this gives only the bitmap's size now then it should probably be `bitmap_size`.